shared/oae/bundles/email/default.properties

```diff
@@ -8,4 +11,8 @@ ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_MULTIPLE_ACTORS = New activity is waiting for y
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_MULTIPLE_ACTIVITIES = ${actorLink} has been very active recently
 ACTIVITY_EMAIL_SUMMARY_IMMEDIATE_SINGLE_ACTOR_SINGLE_ACTIVITY = ${actorLink} has been active recently
 ACTIVITY_EMAIL_SUMMARY_WEEKLY = Here's what happened in the last week
-
+EMAIL_VERIFY_ADMIN_BODY_EXPLANATION = A request to change the email address of your account has been made by an administrator. If you’d like to use this email address instead, please click the following link.
```

Comment:
I find the layout and the wording of the verification email a bit awkward. Can we change the email to the following layout:

1. Logo
2. Large button that says "Verify my email" (similar to one used for Accept this invitation)
3. Small text in grey "John Norman has invited you to collaborate" style that says "We have received a request to change the email address of your account. Please click the button to verify your email address. If you did not request this change, please ignore this message." Note that this text should be in a white container.